### PR TITLE
f-cookie-banner@v4.3.0 - Update f-cookie-banner (and static cookie banner) to Node 16 compatible dependencies

### PR DESCRIPTION
**Pull request by siggerzz**
This PR updates both cookie banner / cookie banner static to have Node 16 compatible dependencies

### CODE REVIEW

**Review by tommcclean — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

File: packages/components/organisms/f-cookie-banner/package.json [hunk 1]
```diff
@@ -51,18 +51,18 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "@justeat/f-services": "1.14.0"
+    "@justeat/f-services": "1.x"
```
Comment (jamieomaguire) [inline packages/components/organisms/f-cookie-banner/package.json:54]: is there any reason this shouldn't be a peer & dev dependency?

File: packages/components/organisms/f-cookie-banner-static/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v1.2.0
+------------------------------
+*July 26, 2022*
+
+### Updated
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-cookie-banner-static/CHANGELOG.md:10]: NIT. Changed.

File: packages/components/organisms/f-cookie-banner/package.json (hunk 1 above)
Comment (siggerzz) [reply packages/components/organisms/f-cookie-banner/package.json:54]: Yeah I was wondering this. It's something I wanted to look into as part of the f-services ticket that's in sprint 👍🏼.

File: packages/components/organisms/f-cookie-banner-static/CHANGELOG.md (hunk 2 above)
Comment (siggerzz) [reply packages/components/organisms/f-cookie-banner-static/CHANGELOG.md:10]: Working in this component in another branch, so will update it then, just to save me having to run another build!

**Pull request merged**